### PR TITLE
Disabling SystemRendering mode for ToolStrip in HC mode to make buttons contrasted

### LINKED ISSUE #1421: Accessibility: [High Contrast]: The text cannot be shown when focus "Close" button of ToolStrip item in Print Preview dialog
* .NET Core Version: 3.0.100 Preview8 13316 from master branch
* Have you experienced this same bug with .NET Framework?: Yes.

**Repro Steps:**

1. Open control panel and select the respective high contrast theme.
2. Open the attached application [PrintDemo .zip](https://github.com/dotnet/winforms/files/3405539/PrintDemo.zip), build and run. 
3. Click the 'Preview' button to show the Print preview dialog then focus on ‘Close’ button.

**Actual:**
The text cannot be shown when focus "Close" button of 'Print Preview' dialog on the Black and White theme. See following screenshot for Black and White theme.
image

**Expected:**
The text should be shown when focus "Close" button of 'Print Preview' dialog on the Black and White theme. See following screenshot for # 1 and # 2 theme.
image

### PR DESCRIPTION
**Pull request by M-Lipin**
…ns contrasted.

Fixes #1421

## Proposed changes

- Disabling System RenderMode for the PrintPreview ToolStrip to prevent ToolStripSystemRenderer to draw buttons with not contrasted color in HC themes.
- The change works and is effective only in HC mode. To enable change the ToolStrip control should be recreated or app should be restarted in HC mode enabled environment.
- In non-HC mode environment RenderMode.System is enabled by default (as it was before)

## Customer Impact

- In HC mode PrintPreview ToolStrip buttons are enough contrasted.
- in non-HC mode PrintPreview ToolStrip buttons are rendered as before the change.
- App should be restarted to make the change effective if system HC mode is turned on/off.

## Regression? 

- No

## Risk

- Minimal

### Before

Please see the #1421 

### After

image

## Test methodology 

- Manual testing.
- Awaiting QA approval.
- HC analyzer - contrast level is sufficient.

## Test environment(s) 

Version: 3.0.100-preview8-012929
Commit: 795f8aeaa8
Runtime Environment:
OS Name: Windows
OS Version: 10.0.18362
OS Platform: Windows
RID: win10-x64
Base Path: C:\Program Files\dotnet\sdk\3.0.100-preview8-012929\
Host (useful for support):
Version: 3.0.0-preview8-27907-09
Commit: fc924dc319
.NET Core SDKs installed:
2.1.700-preview-009601 [C:\Program Files\dotnet\sdk]
2.1.800-preview-009696 [C:\Program Files\dotnet\sdk]
3.0.100-preview8-012929 [C:\Program Files\dotnet\sdk]
.NET Core runtimes installed:
Microsoft.AspNetCore.All 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
Microsoft.AspNetCore.All 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
Microsoft.AspNetCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 3.0.0-preview8.19351.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.NETCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 3.0.0-preview8-27907-09 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.WindowsDesktop.App 3.0.0-preview8-27907-09 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/1642)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewDialog.cs [hunk 1]
```diff
@@ -1248,7 +1248,14 @@ void InitForm()
             separatorToolStripSeparator1,
             closeToolStripButton});
             toolStrip1.Name = "toolStrip1";
-            toolStrip1.RenderMode = System.Windows.Forms.ToolStripRenderMode.System;
+
+            // Note: ToolStripSystemRenderer provides not enough contrasted color
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewDialog.cs]: [nit] a bit more English-sounding:
```suggestion
            // in High Contrast mode the color scheme provided by ToolStripSystemRenderer 
            // is not sufficiently contrast; so disable it.
```

Comment (M-Lipin) [reply src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewDialog.cs]: Thanks!

**Comment by M-Lipin** [comment]
QA passed.

**Comment by leecow** [comment]
tactics approved - preview 9

**Pull request merged**